packages/enhanced_gradients/lib/src/hct_color_tween.dart

```diff
@@ -27,19 +27,31 @@ Color? lerpHct(Color? colorA, Color? colorB, double t) =>
     };
 
 Color _lerpHctNonNullable(Color colorA, Color colorB, double t) {
-  final beginHct = Hct.fromInt(colorA.value);
-  final endHct = Hct.fromInt(colorB.value);
+  final beginHct = Hct.fromInt(_colorToInt(colorA));
+  final endHct = Hct.fromInt(_colorToInt(colorB));
 
-  final opacity = lerpDouble(colorA.opacity, colorB.opacity, t)!;
+  final alpha = lerpDouble(colorA.a, colorB.a, t)!;
   final hue = lerpDegrees(beginHct.hue, endHct.hue, t)!;
   final chroma = lerpDouble(beginHct.chroma, endHct.chroma, t)!;
   final tone = lerpDouble(beginHct.tone, endHct.tone, t)!;
 
   return Color(
     Hct.from(hue, chroma, tone).toInt(),
-  ).withOpacity(opacity);
+  ).withValues(alpha: alpha);
 }
 
-Color _scaleAlpha(Color a, double factor) {
-  return a.withAlpha((a.alpha * factor).round().clamp(0, 255));
+Color _scaleAlpha(Color color, double factor) {
+  return color.withValues(alpha: color.a * factor);
+}
+
+/// [Hct] requires int values, so we need to convert [Color].
+int _colorToInt(Color color) {
```

Comment:
Isn't there a `toARGB32()`?

Reply:
It's on `master` (since `3.28.0-1.0.pre`), but it's not available on `stable` yet

Reply:
Bummer